b2sdk/raw_simulator.py

```diff
@@ -48,6 +50,18 @@
 ]
 
 
+def get_bytes_range(data_bytes, bytes_range):
```

Comment:
needs docstring so that every time someone looks at this class he doesn't need to read through the code to get what it is for